server/server.py

```diff
@@ -180,6 +181,78 @@ def count_verified():
     return jsonify({'verified_count': count})
 
 
+@app.route('/submit/testimonial', methods=['POST'])
+@schema.validate({
+    'properties': {
+        'name': {'type': 'string'},
+        'student_number': {'type': 'number'},
+        'program': {'type': 'string'},
+        'year': {'type': 'string'},
+        'testimonial': {'type': 'string', 'maxLength': 1500},
+        'anonymous': {'type': 'boolean'}
+    },
+    'required': ['name', 'student_number', 'program', 'year', 'testimonial']
+})
+def submit_testimonial():
+    '''
+    Submit a testimonial to be approved by the team
+    '''
+    logger.info('Hit route %s', request.path)
+
+    data = request.get_json()
+
+    logger.debug(data)
+
+    bytes_student_number = int_to_bytes(data['student_number'])
+    bytes_student_number = data['name'].encode('utf-8')
+
+    name = ''
+
+    if data['anonymous'] is True:
+        name = 'Anonymous'
+    else:
+        name = data['name'].capitalize().split(' ')[0]
+
+    testimonial = {
+        'encrypted_name': crypto.encrypt(bytes_student_number),

```

Comment:
Are you supposed to be encrypting the name here?

Reply:
Yes, yes I am